### ISSUE TITLE
Suggestion- zoom the map in the combined report

### ISSUE THREAD

**Issue by cop98** [traccar/traccar-web#1109]
At the moment the combined report shows the map in a small portion of the screen. It would be great to be able to enlarge the map, perhaps by being able to resize the screen below, where the events are displayed

<img width="2550" alt="image" src="https://user-images.githubusercontent.com/49091679/235238129-193f707e-c85f-409f-8355-3f354cf53126.png">

**Comment by tananaev** [comment]
It should set the zoom appropriate to show the whole route.

**Comment by cop98** [comment]
Yes, I misspoke. The zoom works, only that the map uses only 1/3 of the screen. It would be nice to have the ability to zoom in, reducing other functionality

**Comment by MikaVilpo** [comment]
And if you zoom in to check deeper some parts of the trip, it is returned to overall zoom instantly. I would prefer to be able to zoom in and move around.
